data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -38,6 +38,8 @@ MDNS_PUBLISHER_IMAGE=$(image_for mdns-publisher)
 HAPROXY_IMAGE=$(image_for haproxy-router)
 BAREMETAL_RUNTIMECFG_IMAGE=$(image_for baremetal-runtimecfg)
 
+CLUSTER_PROFILE={{.ClusterProfile}}
```

Comment:
You should not need to store this in an environment variable. Use `{{.ClusterProfile}}` directly where you need it.